src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -114,6 +114,9 @@ QtObject:
       isPending:{self.isPending}
     )"""
 
+  proc isInTransactionType(self: ActivityEntry): bool =
+    return self.metadata.activityType == backend.ActivityType.Receive or self.metadata.activityType == backend.ActivityType.Mint
```

Comment:
👍🏼

Reply:
Comment about this: Don't we want this entry to appear in the activity tab of the wallet that triggered the minting? That is, the address that interacted with the contract?